src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle implements Figure {
+    private String color;
+    private double width;
+    private double height;
+
+    public Rectangle(String color, double height, double width) {
+        this.color = color;
+        this.height = height;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return height * width;
+    }
+
+    @Override
```

Comment:
The 'draw' method should not start with an empty line. Please remove the empty line before the method declaration.